Sources/AudioKit/Nodes/Effects/Dynamics/TransientShaper.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  TransientShaper.swift
+//  AudioKit
+//
+//  Created by Evan Murray on 1/6/21.
+//
```

Comment:
Can you update your headers to be the current 1 liner AudioKit headers. I hate these giant, useless, always getting out of date headers.